src/main/java/com/danielagapov/spawn/Controllers/EventController.java

```diff
@@ -123,4 +124,57 @@ public ResponseEntity<List<UserDTO>> getUsersParticipatingInEvent(@PathVariable
             return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
         }
     }
+
+    // full path: /api/v1/events/{eventId}/participating?userId={userid}
+    @GetMapping("events/{eventId}?userId/participating?userId={userid}")
```

Comment:
```suggestion
    @GetMapping("{eventId}?userId/participating?userId={userid}")
```
`events/` is already included in the path, from the top `@RequestMapping("api/v1/events")`

Reply:
I'd suggest you read https://restfulapi.net/resource-naming/ for a guide on how to name this URL, and I'll let you come up with a different approach. Hint: there should never be a need for two '?' + in these Java `@GetMapping()`s, we can leave those request params out and instead have them in the function as arguments, like `(@RequestParam userId)`, as you can see with some of our other endpoints.

Reply:
Also, following your approach of having the `UserParticipationStatus`, you could refactor these two endpoints `isParticipating` and `isInvited` into one that just returns the status enum.